### PR TITLE
SRA Toolkit 2.10.9 gompi/2020b

### PR DESCRIPTION
**Pull request by fizwit**

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 1 out of 2 (1 easyconfigs in total)
node368.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/dcc958211cc85f9478ec28c4264b08a6 for a full test report.

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-goimpi-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,104 @@
+easyblock = 'Bundle'
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-goimpi-2020b.eb:1]: @fizwit please fix the type in the toolchain in the filename - should be `gompi`.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-goimpi-2020b.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-goimpi-2020b.eb]: typo fixed

Comment (verdurin) [reply easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-goimpi-2020b.eb]: Nice that I included a typo in my request to fix a typo...

File: easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-gompi-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+components = [
+    # NCBI-VDB has to be installed as component instead of dependency because SRA-Tools needs its sources
+    ('NCBI-VDB', version, {
+        'patches': ['NCBI-VDB-%(version)s_fix-LD_LIBRARY_PATH.patch'],
+        'checksums': [
+            '3707f81412dbf6ce2b29d3e65e364116c86acc534543e5a084cf8a666b81960d',  # NCBI-VDB-2.10.9.tar.gz
+            # NCBI-VDB-2.10.9_fix-LD_LIBRARY_PATH.patch
+            '054a927baea72d508033974ad6ac072e58aae715d437adb2179b3bc8b62c89d9',
```
Comment (verdurin) [inline easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-2.10.9-gompi-2020b.eb:66]: @fizwit the tests are complaining about the patch checksum - could you update please?

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by fizwit** [comment]
wierd, eb --check-contrib SRA-Toolkit-2.10.9-gompi-2020b.eb  did not find the issue with the checksum.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node308.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/73cdc438d79c1845f3c0623e5f398ec4 for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @fizwit !

**Pull request merged**